### DISCUSSION TITLE
how to tell which module to use which auth?

### DISCUSSION THREAD

**Discussion by ibigbug** [prometheus/snmp_exporter#1037] [Q&A]
after migrating to the new config format https://github.com/prometheus/snmp_exporter/blob/main/generator/FORMAT.md

how to tell which module to use which auth defined in the `auths` section?

**Comment by SuperQ** [comment]
You do this in the Prometheus config via `params`. Modules are no longer tied to specific auths. Any module can be used with any auth.

**Reply by ibigbug** [reply]
thank you @SuperQ 
i this there's  example:

```
    params:
      auth: [public_v2]
      module: [if_mib]
```

and i want to have module v2 to use auth_v2 and module v3 to use auth_v3, do you have a sample how I can config the params that way?
